lib/src/client/call.dart

```diff
@@ -352,3 +367,23 @@ class ClientCall<Q, R> implements Response {
     } catch (_) {}
   }
 }
+
+/// Given a string, ensure the length is a multiple of 4, adding padding if needed.
+String padBase64Multiple4(String input) {
+  var data = input ?? '';
+  while (data.length % 4 != 0) {
+    data += '=';
+  }
+  return data;
+}
+
+List<GeneratedMessage> decodeStatusDetails(String data) {
+  try {
+    /// Parse each Any type into the correct GeneratedMessage
```

Comment:
Consider making this doc comment a doc comment for the whole function - rather than doc comment here.

Reply:
I think this comment still applies.